lua/entities/gmod_wire_turret.lua

```diff
@@ -1,182 +1,219 @@
+	end
+
+	-- Shoot a bullet
+	local bullet      = {}
+	bullet.Num        = self.numbullets
+	bullet.Src        = shootOrigin
+	bullet.Dir        = shootAngles:Forward()
+	bullet.Spread     = self.spreadvector
+	bullet.Tracer     = self.tracernum
+	bullet.TracerName = self.tracer
+	bullet.Force      = self.force
+	bullet.Damage     = self.damage
+	bullet.Attacker   = self:GetPlayer()
+	bullet.BlastDamage   = self.blastdamage
+	bullet.BlastRadius = self.blastradius
+	bullet.Callback   = function(attacker, traceres, cdamageinfo)
+		WireLib.TriggerOutput(self, "HitEntity", traceres.Entity)
+		return BlastTrace(0, self, self:GetPlayer(), traceres, self.blastdamage, self.blastradius)
+	end
+
+	self:FireBullets( bullet )
+
+	-- Make a muzzle flash
+	self.effectdata:SetOrigin( shootOrigin )
+	self.effectdata:SetAngles( shootAngles )
+	self.effectdata:SetScale( 1 )
+	util.Effect( "MuzzleEffect", self.effectdata )
+end
+
+function BlastTrace(num, self, attacker, tr, dmg, radius)
+	if (num > 1) then return end
+
+	util.BlastDamage(self, attacker, tr.HitPos, radius, dmg)
+end
+
+function ENT:OnTakeDamage( dmginfo )
+	self:TakePhysicsDamage( dmginfo )
+end
+
+function ENT:Think()
+	BaseClass.Think( self )
+
+	if ( self.Firing ) then
+		self:FireShot()
+	end
+
+	self:NextThink( CurTime() )
+	return true
+end
+
+local ValidTracers = {
+	["Tracer"]                = true,
+	["AR2Tracer"]             = true,
+	["ToolTracer"]            = true,
+	["GaussTracer"]           = true,
+	["LaserTracer"]           = true,
+	["StriderTracer"]         = true,
+	["GunshipTracer"]         = true,
+	["HelicopterTracer"]      = true,
+	["AirboatGunTracer"]      = true,
+	["AirboatGunHeavyTracer"] = true,
+	[""]                      = true
+}
+
+function ENT:SetSound( sound )
+	sound = string.Trim( tostring( sound or "" ) ) -- Remove whitespace ( manual )
+	local check = string.find( sound, "[\"?]" ) -- Preventing client crashes
+	self.sound = check == nil and sound ~= "" and sound or nil -- Apply the pattern check
+end
+
+function ENT:SetDelay( delay )
+	local check = game.SinglePlayer() -- clamp delay if it's not single player
+	local limit = check and 0.01 or 0.05
+	self.delay = math.Clamp( delay, limit, 1 )
+end
+
+function ENT:SetNumBullets( numbullets )
+	local check = game.SinglePlayer() -- clamp num bullets if it's not single player
+	local limit = math.floor( math.max( 1, numbullets ) )
+	self.numbullets = check and limit or math.Clamp( limit, 1, 10 )
+end
+
+function ENT:SetTracer( tracer )
+	local tracer = string.Trim(tracer)
+	self.tracer = ValidTracers[tracer] and tracer or ""
+end
+
+function ENT:SetSpread( spread )
+	self.spread = math.Clamp( spread, 0, 1 )
+	self.spreadvector.x = self.spread
+	self.spreadvector.y = self.spread
+end
+
+function ENT:SetDamage( damage )
+	self.damage = math.Clamp( damage, 0, 100 )
+end
+
+function ENT:SetBlastDamage( blastdamage )
+	self.blastdamage = math.Clamp( blastdamage, 0, GetConVar("wire_turret_blastdamage_max") )

```

Comment:
`GetConVar` returns the actual `ConVar` object. You want either `GetConVar(...):GetInt()` or `GetConVarInt(...)`.

Ideally you'd keep the convar outside of the function, and then just call GetInt on that whenever SetBlastDamage is called